src/constants/tokens.ts

```diff
@@ -44,6 +44,11 @@ export const GitcoinStakedETHIndex: Token = {
   symbol: 'gtcETH',
 }
 
+export const HighYieldETHIndex: Token = {
+  address: '0xc4506022Fb8090774E8A628d5084EED61D9B99Ee',
```

Comment:
to discuss later: adding `@indexcoop/tokenlists` as a dep and importing these constants from there

Reply:
Task already available here: https://github.com/IndexCoop/flash-mint-sdk/issues/56